app/src/main/assets/malaysia.ics

```diff
@@ -320,45 +321,60 @@ RRULE:FREQ=YEARLY;INTERVAL=1
 SUMMARY:Federal Territory Day
 END:VEVENT
 BEGIN:VEVENT
-DTSTART;VALUE=DATE:20210720
-DTEND;VALUE=DATE:20210721
+DTSTART;VALUE=DATE:20230629
+DTEND;VALUE=DATE:20230630
 UID:mal_52
 STATUS:CONFIRMED
 SUMMARY:Hari Raya Haji
 END:VEVENT
 BEGIN:VEVENT
-DTSTART;VALUE=DATE:20210514
-DTEND;VALUE=DATE:20210515
+DTSTART;VALUE=DATE:20230422
+DTEND;VALUE=DATE:20230425
 UID:mal_53
 STATUS:CONFIRMED
-SUMMARY:Hari Raya Aidilfitri Holiday
+SUMMARY:Hari Raya Aidilfitri
 END:VEVENT
 BEGIN:VEVENT
-DTSTART;VALUE=DATE:20210526
-DTEND;VALUE=DATE:20210527
+DTSTART;VALUE=DATE:20230504
+DTEND;VALUE=DATE:20230505
 UID:mal_56
 STATUS:CONFIRMED
 SUMMARY:Wesak Day
 END:VEVENT
 BEGIN:VEVENT
-DTSTART;VALUE=DATE:20210430
-DTEND;VALUE=DATE:20210501
+DTSTART;VALUE=DATE:20230409
+DTEND;VALUE=DATE:20230410
 UID:mal_57
 STATUS:CONFIRMED
-SUMMARY:Nuzul Al-Quran observed (regional holiday)
+SUMMARY:Nuzul Al-Quran (Kelantan\, Terengganu)
 END:VEVENT
 BEGIN:VEVENT
-DTSTART;VALUE=DATE:20210620
-DTEND;VALUE=DATE:20210621
+DTSTART;VALUE=DATE:20230618
+DTEND;VALUE=DATE:20230619
 UID:mal_58
 STATUS:CONFIRMED
 SUMMARY:Birthday of the Sultan of Kedah
 END:VEVENT
 BEGIN:VEVENT
-DTSTART;VALUE=DATE:20210719
-DTEND;VALUE=DATE:20210720
+DTSTART;VALUE=DATE:20230627
+DTEND;VALUE=DATE:20230628
 UID:mal_59
 STATUS:CONFIRMED
 SUMMARY:Arafat Day (Terengganu)
 END:VEVENT
 END:VCALENDAR
```

Comment:
the file has to end with this END:VCALENDAR and not END:VEVENT

Reply:
fixed